resources/qml/LabelBar.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.1
+import QtQuick.Layouts 1.3
+import UM 1.2 as UM
+// The labelBar shows a set of labels that are evenly spaced from oneother.
+// The first item is aligned to the left, the last is aligned to the right.
+// It's intended to be used together with RadioCheckBar. As such, it needs
+// to know what the used itemSize is, so it can ensure the labels are aligned correctly.
+Item
+{
+    id: base
+    property var model: null
+    property string modelKey: ""
+    property int itemSize: 14
+    height: childrenRect.height
+    RowLayout
+    {
+        anchors.left: parent.left
+        anchors.right: parent.right
+        height: label.height
```

Comment:
I got this QML error: `resources/qml/LabelBar.qml:20: ReferenceError: label is not defined`